CONTRIBUTING.md

```diff
@@ -1,63 +1,63 @@
 
 ## Testing the Provider
 
-To run the go tests and check test coverage run the following:
+To run the Go tests and check test coverage run the following:
 
 ```sh
-$ go test -v ./... -cover
+go test -v ./... -cover
 ```
 
 ## Generating Documentation
 
-Documentation is generated for the provider using the [terraform documentation plugin](https://github.com/hashicorp/terraform-plugin-docs). This plugin uses the schema `Description` field in conjunction with the contents of the `/templates` and `/examples` folders to generate the `/docs` content.
+Documentation is generated for the provider using [terraform-plugin-docs](https://github.com/hashicorp/terraform-plugin-docs). This plugin uses the schema `Description` field in conjunction with the contents of the `/templates` and `/examples` folders to generate the `/docs` content.
 
-To regenerate the `/docs` markdown run:
+To regenerate the `/docs` Markdown run:
 
 ```sh
-$ go generate
+go generate
 ```
 
-## Installing Locally
+## Installing plugin locally
 
-To install the binary it must be copied to the appropriate terraform plugin directory. You may need to create the appropriate directories. For more information [see these Terraform 0.13 plugin docs](https://www.hashicorp.com/blog/automatic-installation-of-third-party-providers-with-terraform-0-13)
+To install the binary it must be copied to the appropriate Terraform plugin directory (note that you may need to create these directories first). You can refer to the [Terraform 0.13 plugin docs](https://www.hashicorp.com/blog/automatic-installation-of-third-party-providers-with-terraform-0-13) for more information.
 
-The current version of the provider is considered to be 0.2 and `darwin_amd64` should match your machines operating system and architecture in the format `$OS_$ARCH`. For example macOS is `darwin_amd64` and linux is `linux_amd64`.
+The current version of the provider is considered to be 0.2 and `darwin_amd64` should match your machines operating system and architecture in the format `$OS_$ARCH`. For example, macOS is `darwin_amd64` and Linux is `linux_amd64`.
 
 ```sh
-$ mkdir -p ~/.terraform.d/plugins/github.com/1Password/onepassword/0.2/darwin_amd64/
-$ cp ./terraform-provider-onepassword ~/.terraform.d/plugins/github.com/1Password/onepassword/0.2/darwin_amd64/terraform-provider-onepassword
+mkdir -p ~/.terraform.d/plugins/github.com/1Password/onepassword/0.2/darwin_amd64/
+cp ./terraform-provider-onepassword ~/.terraform.d/plugins/github.com/1Password/onepassword/0.2/darwin_amd64/terraform-provider-onepassword
 ```
 
 ## Using plugin locally
 
-In your Terraform configuration you will need to specify the op plugin with:
+In your Terraform configuration you will need to specify the `op` plugin with:
 
 ```tf
 terraform {
   required_providers {
     onepassword = {
-      source   = "github.com/1Password/onepassword"
-      version = "~> 1.0.0"
+      source  = "1Password/onepassword"
```

Comment:
FYI for reviewers: both the `README.md` and `examples/main.tf` files use this syntax for the value of `source`, so I dropped the `github.com` domain here too so that everything is consistent.

Reply:
Then the better question for this is: Does it actually work this way? 🤔 
I assume that it does, but it's better to double check.

Reply:
I can confirm that it works if setting the source as `"1Password/onepassword"`